sale_product_set_packaging_qty/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Sale Product Set Packaging Quantity",
+    "summary": "Manage packaging and quantities on product set lines",
+    "version": "14.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Warehouse Management",
+    "website": "https://github.com/OCA/sale-workflow",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": ["sale_product_set"],
```

Comment:
I think there is a missing dependency on sale_stock. On trying to add a product set with packaging:
![Screenshot from 2021-03-26 16-39-07](https://user-images.githubusercontent.com/19902174/112656630-e0cda200-8e51-11eb-812c-c6c9161a8948.png)

Reply:
Apart from that LGTM

Reply:
@phuctranerp ping

Reply:
sorry for late reply, I missed the message. fixed in the latest update. Thanks

Reply:
Will you fix 13.0 too ?